pyproject.toml

```diff
@@ -1,6 +1,6 @@
 [tool.poetry]
 name = "xrpl-py"
-version = "3.0.0"
+version = "4.0.0"
```

Comment:
should this be a beta version with bX postfix?

Reply:
Nope, it should be normal major release since MPT is available in devnet and rippled development is completed.